app/controllers/resources_controller.rb

```diff
@@ -6,12 +6,29 @@ def index
                 else
                   Resource.all
                 end
-
-    render json: resources.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    render json: resources.to_json(RENDERED_RESOURCE_ATTRIBUTES)
   end
 
   def show
     resource = Resource.find(params[:id])
-    render json: resource.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    render json: resource.to_json(RENDERED_RESOURCE_ATTRIBUTES)
   end
+
+  # rubocop:disable MutableConstant
```

Comment:
Ah, thanks for catching this. I haven't looked at what `to_json` does under the hood to mutate the hash, but it could conceivably alter it in such a way that subsequent calls behave in unexpected ways. I'm making some changes now to improve the performance of the resource index endpoint; while I'm at it I'll move this stuff to a method instead so we get a fresh hash every time. Seems like bad practice for `to_json` to mutate the argument though :-/.